docs/Developing.md

```diff
@@ -132,14 +132,81 @@ or visit `http://localhost:3000/ui/` direct.
 
 xpcshell and mochitest
 ----------------------
-
 These are tests for the [chrome](https://developer.mozilla.org/docs/Glossary/Chrome)
 modules and files.
 
-Currently these are not runnable in the github repository, and an export to
-mozilla-central is required. We are
-[working on improving](https://bugzilla.mozilla.org/show_bug.cgi?id=1238570) this
-situation.
+**Test Export to Mozilla Central(gecko-dev)**
+
+The gecko-dev repo is cloned locally and is used as the xpcshell and 
+mochitest test build.
+
+**Resetting gecko-dev repo for pull and export**
+
+If gecko-dev has already been cloned and was used for a previous export and build,
+we can reset the repo:
+```shell
+gecko-dev$ git checkout fx-team
+gecko-dev$ git checkout -- .
+gecko-dev$ git clean -f
+gecko-dev$ git status
+``` 
+Once reset, you should not see gecko-dev/browser/extensions/loop/ directory or files.
+You can now pull fx-team latest.
+```shell
+gecko-dev$ git pull
+```  
+
+**Git export from loop repo**
+
+Export of the add-on code changes from loop to gecko-dev is done through loop makeFile.
+A loop checked out branch that is rebased to latest master can be used or master can slso be used
+for export.
+```shell
+loop$ make git-export
```

Comment:
Actually `git-export` is quite bogus. It doesn't rely on git at all. So whilst we're here, lets change this to `export_mc`, and then we're not implying the type.